client/app/account/billing/autoRenew/billing-autoRenew.html

```diff
@@ -22,14 +22,6 @@ <h1 data-translate="autorenew_title"></h1>
                     data-dismiss="alert">
             </button>
             <p data-ng-if="!automaticRenewV2Mean.allowed" data-translate="autorenew_service_renew_requires_mean_and_date"></p>
-            <p data-ng-if="automaticRenewV2Mean.allowed && !nicRenew.active">
-                <span data-translate="autorenew_service_activate_alert"></span>
-                <button class="btn btn-default"
-                        type="button"
-                        data-ng-click="setAction('active', { nicRenew: nicRenew }, 'autoRenew')"
```

Comment:
I think now there are no more `setAction('active')` which means that these files aren't reachable (modal):

`client/app/account/billing/autoRenew/active/billing-autoRenew-active.controller.js`
`client/app/account/billing/autoRenew/active/billing-autoRenew-active.html`

Could you please confirm?